velox/dwio/parquet/writer/Writer.h

```diff
@@ -79,11 +92,14 @@ class Writer {
       memory::MemoryPool& pool,
       int32_t rowsInRowGroup,
       std::shared_ptr<::parquet::WriterProperties> properties =
-          ::parquet::WriterProperties::Builder().build())
+          ::parquet::WriterProperties::Builder().build(),
+      std::shared_ptr<velox::core::QueryCtx> queryCtx =
```

Comment:
You don't need the whole `QueryCtx`, just pass grow ratio (or an `Options` object to be future proof) here is enough

Reply:
Do you mean to pass the value of grow ratio here, such as 1? If so, is there no need to add the configuration of `kDataBufferGrowRatio `in `QueryConf.h` file?

Reply:
You should receive it from connector config in `HiveDataSink`.  When you create `QueryCtx` in the engine code you will pass a list of connector configs.

Reply:
It seems that we can't get this config in `HiveDataSink`. Because `HiveDataSink `only supports `dwrf::writer` and not `parquet::writer`. My current approach is to only pass in a grow ratio.